DQM/DTMonitorModule/src/DTTimeEvolutionHisto.cc

```diff
@@ -166,7 +166,7 @@ void DTTimeEvolutionHisto::updateTimeSlot(int ls, int nEventsInLS) {
       }
 
       // Set only labels which can be seen in the plot without zooming
-      if (binLabelCounter % ((int)(nBookedBins / 25)) == 0)  //around 25 labels fit in a full size hist
+      if (nBookedBins < 25 || binLabelCounter % (nBookedBins / 25) == 0)  //around 25 labels fit in a full size hist
```

Comment:
@iarspider , shouldn't this be `nBookedBins >= 25 &&` ?

Reply:
I think it can be either one. If we go `&&` route, the label will not be set for `nBookedBins < 25`.

Reply:
if `nBookedBins<25` then we should have seen floating point exception (dueto modulo zero) ... right? If this code has not generated floating point exception so I guess `nBookedBins` is `>=25`,  so I am fine with either `assert(nBookedBins>=25)` or `nBookedBins >= 25 &&`

Reply:
The condition will short-circuit, i.e. if left hand side of `||` is true then the right hand side will not be evaluated.

Reply:
@jfernan2 noted, thanks!